omgeo/services/google.py

```diff
@@ -15,13 +16,20 @@ class Google(GeocodeService):
 
     DEFAULT_PREPROCESSORS = [ComposeSingleLine()]
 
+    DEFAULT_POSTPROCESSORS = [
+        AttrListExcludes([
+            'establishment',  # In practice this seems to be used for people / businesses
```

Comment:
I see this is listed under "deprecated types" in the [Places documentation](https://developers.google.com/places/supported_types). Under the [geocoding documentation](https://developers.google.com/maps/documentation/geocoding/intro) `establishment` is described as "typically indicates a place that has not yet been categorized."

What led to using it here? Were there any other, better sources of documentation? It would be helpful to expand on the comment here, to make it clear that this is to exclude from Google geocoder results both results for businesses, and also people *associated with businesses*.

Reply:
I was mostly looking at the geocoding documentation you linked, and unfortunately couldn't find anything better. I arrived at excluding `establishment` via experimenting with a wide variety of different geocoding searches.